crates/oxc_parser/src/lexer/simd.rs

```diff
@@ -39,23 +39,25 @@ impl SkipWhitespace {
     }
 
     pub fn simd(&mut self, bytes: &[u8]) -> &Self {
-        let (chunks, remainder) = bytes.as_chunks::<ELEMENTS>();
+        let chunks = bytes.chunks(ELEMENTS);
 
         for chunk in chunks {
-            self.check_chunk(chunk);
-            if self.found {
-                return self;
+            if chunk.len() != ELEMENTS {
```

Comment:
@Boshen For the unstable `slice.as_chunks` api, element of `chunks` in `(chunks, remainder)` could also have a length less than `ELEMENTS`, if `bytes.len()<ELEMENTS`.

So, in the for loop, there is a case that a `chunk` could have length less than `ELEMENTS`. Is that as expected?